ui/components/app/account-menu/account-menu.component.js

```diff
@@ -428,7 +428,7 @@ export default class AccountMenu extends Component {
           }}
           icon={
             <IconCog
-              color="var(--color-icon-default)"
+              color="var(--color-icon-alternative)"
               ariaLabel={t('settings')}
             />
           }
```

Comment:
Updating these icons colors actually keeps their color the same.

<img width="363" alt="Screen Shot 2022-05-17 at 2 30 19 PM" src="https://user-images.githubusercontent.com/8112138/168913415-b7453389-094a-4935-aafa-4ce76d1c5226.png">